src/components/ProtectedRoute/ProtectedRoute.js

```diff
@@ -0,0 +1,24 @@
+import Cookies from 'universal-cookie';
+import React from 'react';
+import { Route, Redirect } from 'react-router-dom';
+const cookies = new Cookies();
+export default function PrivateRoute({ component: Component, ...rest }) {
```

Comment:
Why the name is PrivateRoute?

Reply:
@akshatnitd I have changed the name to "ProtectedRoute".
I think the PR is fine now.